pyEpiabm/pyEpiabm/sweep/spatial_sweep.py

```diff
@@ -205,8 +207,11 @@ def find_infectees_Covidsim(self, infector: Person,
             # Sample at random from the infectee cell to find
             # an infectee
             infectee = random.sample(infectee_cell.persons, 1)[0]
-            infection_distance = (DistanceFunctions.dist(
-                infector.microcell.cell.location, infectee_cell.location) /
+            # Covidsim used explicitly the kernel distance between people,
+            # and divided by the kernel of the minimum distance between
+            # their cells.
```

Comment:
I would clarify this comment and what you mean by minimum distance

Reply:
Not sure if my rewrite helped, let me know if you want it further explained